be/src/storage/compaction_manager.h

```diff
@@ -38,12 +38,11 @@ class CompactionManager {
 
     void update_candidates(std::vector<CompactionCandidate> candidates);
 
```

Comment:
I think the CompactionManager and CompactionScheduler may have some overlaps.

Reply:
it can be unify to one class, but if we want to support multi scheduler, it may be useful later.